core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3242,6 +3237,7 @@ public void cleanUp() {
             if (JenkinsJVM.isJenkinsJVM()) {
                 JenkinsJVMAccess._setJenkinsJVM(oldJenkinsJVM);
             }
+            ClassFilterImpl.unregister();
```

Comment:
Without this line, a bunch of tests will fail, because a stale `ClassFilterImpl` will be registered in the system. Worse, individual test cases will pass but the containing test suites will fail, which is rather confusing.

If there is any ongoing deserialization in the system, that would certainly be a bug. Most stuff running at this point would fail anyway due to `Jenkins.getInstance()` calls. At worst we are back to the standard blacklist in effect prior to this JEP.

Reply:
OK, fingers crossed